docs/source/internals/getting_started.rst

```diff
@@ -59,14 +59,11 @@ context processors.
 
 .. code-block:: django
 
-    from oscar import OSCAR_MAIN_TEMPLATE_DIR
-
     TEMPLATES = [
         {
             'BACKEND': 'django.template.backends.django.DjangoTemplates',
             'DIRS': [
                 os.path.join(BASE_DIR, 'templates'),
```

Comment:
wouldn't it be nicer to just add "oscar" to INSTALLED_APPS and have no template configuration at all?

Reply:
I guess this is what is going to be achieved by this change

Reply:
No I mean the whole ``TEMPLATES =`` thing can be removed from the documentation because it is not needed.

Reply:
oh yeah, it's not needed anymore!! so should be dropped as well